react/index.js

```diff
@@ -57,79 +39,35 @@ class Header extends Component {
     return acceptedPaths.test(page)
   }
 
-  componentWillUnmount() {
-    document.removeEventListener('scroll', this.handleScroll)
-  }
-
-  handleScroll = () => {
-    if (!this._root.current) {
-      return
-    }
-
-    const scroll = window.scrollY
-    const { scrollHeight } = this._root.current
-
-    const showMenuPopup = scroll >= scrollHeight
-
-    if (showMenuPopup !== this.state.showMenuPopup) {
-      this.setState({
-        showMenuPopup
-      })
-    }
-  }
-
   render() {
-    const { linkUrl, logoUrl, logoTitle, orderFormContext, showSearchBar, showLogin } = this.props
-    const { showMenuPopup } = this.state
+    const { logoUrl, linkUrl, logoTitle, showSearchBar, showLogin } = this.props
 
     const leanMode = this.isLeanMode()
-    const offsetTop = (this._root.current && this._root.current.offsetTop) || 0
-
-    const hasMessage =
-      orderFormContext.message.text && orderFormContext.message.text !== ''
 
     const topMenuOptions = {
       linkUrl,
       logoUrl,
       logoTitle,
-      leanMode,
       showSearchBar,
       showLogin,
     }
 
     return (
-      <Fragment>
-        <ExtensionPoint id="telemarketing" />
-        <div
-          className={`vtex-header force-full-width relative z-2 w-100 bb bw1 b--muted-4 ${leanMode ? 'vtex-header-lean-mode' : ''}`}
-          ref={this._root}
-        >
-          <div className="z-2 items-center w-100 top-0 bg-base tl">
-            <ExtensionPoint id="menu-link" />
-          </div>
-          <TopMenu {...topMenuOptions} />
-          {!leanMode && <ExtensionPoint id="category-menu" />}
-          <div style={{ visibility: showMenuPopup ? 'inherit' : 'hidden' }}>
-            <TopMenu fixed {...topMenuOptions} />
-          </div>
-          <div
-            className="flex flex-column items-center fixed w-100"
-            style={{ top: offsetTop + 120 }}
-          >
-            {hasMessage && (
-              <div className="pa2 mw9">
-                <Alert
-                  type={
-                    orderFormContext.message.isSuccess ? 'success' : 'error'
-                  }
-                >
-                  {orderFormContext.message.text}
-                </Alert>
-              </div>
-            )}
-          </div>
-        </div>
-      </Fragment>
+      <div
+        className={`vtex-header force-full-width relative z-2 ${leanMode ? 'vtex-header-lean-mode' : ''}`}
```

Comment:
maybe use `classnames`?

Reply:
@lucasecdb I'm not really into adding the `classNames` dependency if it's just going to be used here and there. I can add it if you insist though, I'm not really against it